docs/auto-setup.adoc

```diff
@@ -135,3 +133,16 @@ or, destroy the machine:
 vagrant destroy
 ```
 and run it again.
+
+
+== Troubleshooting
+
+[#ssh-port]
+=== SSH Port was not properly retrieved from SSHConfig
+
+If you see an error `vagrant: error: SSH Port was not properly retrieved from
+SSHConfig.` or `vagrant: strconv.Atoi: parsing "": invalid syntax` try to:
+
+- Turn off your antivirus completely. You can also try to disable the SSL scanning in your antivirus options.
+
+- If that doesn't help (as may happen after update to MacOS Big Sur), execute `csrutil enable && reboot` in the Recovery Mode (as described in this https://apple.stackexchange.com/a/410145[Stack Exchange answer])
```

Comment:
Maybe we could also add the info about the command that debugs the `packer build` command, because it helped us with finding what is the problem. For example: 

"If all of the above fails you can also use 

`PACKER_LOG=1 packer build -only=vagrant packerfile.json`

to debug the building process and see the exact issue."

Reply:
Thanks, added